pygeos/constructive.py

```diff
@@ -703,3 +704,28 @@ def voronoi_polygons(
     <pygeos.Geometry GEOMETRYCOLLECTION EMPTY>
     """
     return lib.voronoi_polygons(geometry, tolerance, extend_to, only_edges, **kwargs)
+
+
+@requires_geos("3.8.0")
+@multithreading_enabled
+def minimum_bounding_circle(geometry, **kwargs):
+    """Computes the minimum bounding circle that encloses an input geometry.
+
+    Parameters
+    ----------
+    geometry : Geometry or array_like
+
+    Examples
+    --------
+    >>> minimum_bounding_circle(Geometry("POLYGON ((0 0, 0 10, 10 10, 10 0, 0 0))"))
+    <pygeos.Geometry POLYGON ((12.1 5, 11.9 3.62, 11.5 2.29, 10.9 1.07, 10 3.55e...>
+    >>> minimum_bounding_circle(Geometry("LINESTRING (1 1, 10 10)"))
+    <pygeos.Geometry POLYGON ((11.9 5.5, 11.7 4.26, 11.4 3.06, 10.8 1.96, 10 1, ...>
+    >>> minimum_bounding_circle(Geometry("MULTIPOINT (2 2, 4 2)"))
+    <pygeos.Geometry POLYGON ((4 2, 3.98 1.8, 3.92 1.62, 3.83 1.44, 3.71 1.29, 3...>
+    >>> minimum_bounding_circle(Geometry("POINT (0 1)"))
+    <pygeos.Geometry POINT (0 1)>
+    >>> minimum_bounding_circle(Geometry("GEOMETRYCOLLECTION EMPTY"))
+    <pygeos.Geometry POLYGON EMPTY>
```

Comment:
Thanks; same comment as the other PR, can you include these as unittests as well?